src/map/script-fun.cpp

```diff
@@ -2175,6 +2175,30 @@ void builtin_pvpoff(ScriptState *st)
     }
 }
 
+static
+void builtin_killer(ScriptState *st)
+{
+    dumb_ptr<map_session_data> sd = script_rid2sd(st);
+    sd->state.killer = !sd->state.killer;
+
+    if (sd->state.killer)
```

Comment:
@wushin can you please make this use arg0 so that we can do "killer 0;" and "killer 1;" ?
same for killable

:heart: